slices/vim.yaml

```diff
@@ -0,0 +1,24 @@
+package: vim
+
+essential:
+  - vim_copyright
+
+slices:
+  bins:
+    essential:
+      - libacl1_libs
+      - libc6_libs
+      - libgpm2_libs
+      - libpython3.12t64_libs
+      - libselinux1_libs
+      - libsodium23_libs
+      - libtinfo6_libs
+      - vim-common_config
+      - vim-runtime_plugins
+    contents:
+      /usr/bin/vim: {symlink: /usr/bin/vim.basic}
```

Comment:
This is fine, until we have packages like `vim-gtk3` coming in where it demands with a higher priority that `vim` should point to `vim.gtk3`:

```
update-alternatives --install /usr/bin/vim vim /usr/bin/vim.gtk3 50
```

Thoughts on how we should address the `vi` and `vim` symlinks?